packages/experiments/src/components/Pagination/examples/Pagination.Buttons.Customization.Round.Example.tsx

```diff
@@ -49,8 +50,14 @@ export class PaginationCustomizationRoundExample extends React.Component<{}, IPa
                 borderRadius: '16px',
                 color: 'white'
               },
-              '&:hover:aria-selected=true': { color: 'white' },
-              '&:hover[aria-selected=true]': { color: 'white' }
+              '&:hover:aria-selected=true': {
```

Comment:
I believe you can remove this selector as it is `invalid` without the brackets which you've redefined below.

Reply:
removed

---
In reply to: [241932197](https://github.com/OfficeDev/office-ui-fabric-react/pull/7411#discussion_r241932197) [](ancestors = 241932197)